napari_console/qt_console.py

```diff
@@ -125,18 +125,13 @@ def __init__(self, viewer: 'napari.viewer.Viewer'):
             self.push = lambda var: None
 
         elif isinstance(shell, ZMQInteractiveShell):
-            # if launching from jupyter notebook, connect to the existing
-            # kernel
-            kernel_client = QtKernelClient(
-                connection_file=get_connection_file()
-            )
-            kernel_client.load_connection_file()
-            kernel_client.start_channels()
-
+            # if launching from jupyter notebook, then adding a console is
```

Comment:
Optional: the code is identical and the comment is very similar to the iPython case above. I wonder if we should just merge the condition (i.e. use `isinstance(shell, (TerminalInteractiveShell, ZMQInteractiveShell)`)?

Reply:
Makes sense to me, it's more elegant for sure.
Will do, hopefully later tonight.